packages/design-system/src/styles/components/_UsaBanner.scss

```diff
@@ -32,22 +32,24 @@
 .ds-c-usa-banner__header {
   // Applying display:grid to account for flag icon alignment with text elements in mobile/table viewports
   display: grid;
-  gap: 0.5ch 1ch;
+  gap: 2px 1ch;
   grid-template-areas:
     'icon text'
-    'icon action';
+    'spacer action';
   grid-template-columns: 1.5em auto;
+  grid-template-rows: min-content min-content;
   line-height: var(--heading__line-height);
   min-height: 3rem;
-  padding-block: 0.25rem;
+  padding-block: 0.5rem;
   position: relative;
 
   // Need to target viewport 1px larger than $media-width-md to ensure the layout is correct on the breakpoint
-  @media (max-width: ($media-width-sm + 1)) {
+  @media (max-width: ($media-width-md + 1)) {
```

Comment:
squashed a bug - the comment referenced a md breakpoint and this query was targeting a sm breakpoint. previously caused the text to get overlapped by the close button at a specific breakpoint and this fixes it.